HOL-INS April Rework

KrasnaMerah opened the pull request:
Rectifying stuffs from Bitt3rsteel's latest video

Done
- INS only have Gouvernmentsmarine as its escort MIO ✅
- Add Parindra's flag✅
- Import EIR's ship names ✅
- Nusantara-ise the Malay formable's name ✅
- missing Chief of Navy while Van Dulm retired (should've been Helfrich)✅️
- Rename "Colonial Economy" to "De Malaise"✅️
- More character for the Netherlands and Indonesia, including Prinses Juliana and Margono DHK✅️
- Actually making Monarchist Indonesia viable✅️
- Import EIR's decisions to nationalize the Dutch MIOs✅️
- Import EIR's decisions to establish the experimental facilities✅️

Ready to review, i'll fix the rest for June BETA.

KrasnaMerah committed:
HOL-INS rework, rectifying bugs from Bitt3rsteel's video

Please refer to my bug list stated on my modding nest

KrasnaMerah committed:
220425 update

Juliana's portrait

KrasnaMerah committed:
Update 240425

Fixing the monarchists

marijn211 reviewed (dismissed):
That Wilhelmina event should be history-file-fired, and probably not show when theyre communist

KrasnaMerah committed:
Merge remote-tracking branch 'upstream/Beta-Edits' into East-Indies-Reworked

KrasnaMerah committed:
010525 Update - More advisors and locs

KrasnaMerah committed:
Update 010525 pt2 - Queen event

Now should be working as intended

KrasnaMerah committed:
Update 020525 - Decision and more advisors for INS

Also proper portrait pathing through interface file....

KrasnaMerah committed:
Merge remote-tracking branch 'upstream/Beta-Edits' into East-Indies-Reworked

marijn211 commented:
Previous comment hasnt been completely addressed yet, date > 1948.09.06 should be replaced with firing the event in the history file

KrasnaMerah committed:
God save the Queen, mhm

Reworking the event to be triggered after four thousand days.

KrasnaMerah commented:
Someone pls accept, at least the bugfix parts.

marijn211 committed:
Merge branch 'Beta-Edits' into East-Indies-Reworked

BiccyThiccy committed:
Merge branch 'Beta-Edits' into East-Indies-Reworked

BiccyThiccy committed:
People get shot for these kind of mistakes 👿

@KrasnaMerah
- Fixed Johan van Oorschot being unlocalized
- Deleted two duplicate recruit_characters (I think from someone else merging beta into this)
- Fixed HOL's Naval Expansion not transferring Conrad Helfrich as intended, and instead spamming the error log.
- Gave the Special Project facility decisions icons, and reworked them to move elsewhere if no state is available; iirc they are limited to 1 per state so the way you had it could cause problems.

BiccyThiccy reviewed (approved):
Ready fready 😈
